### PR TITLE
Setup cap deploy

### PR DESCRIPTION
**Pull request by devanshu-m**
Cap deploy for dlp-curate

Connected to #29

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -1,6 +1,7 @@
 source 'https://rubygems.org'
 
-ruby '2.4.2'
+
+ruby '>=2.4.2', '<=2.5.99'
```
Comment (devanshu-m) [inline Gemfile]: Do we want a specific ruby version or are we ok with a range?

Comment (rotated8) [reply Gemfile]: I believe 2.6 will come out soon as well, so 2.4 may be reaching end of support, and we'll need to move up to 2.5 anyway.

File: Gemfile [hunk 2]
```diff
@@ -52,6 +55,17 @@ group :development do
   gem 'spring-watcher-listen', '~> 2.0.0'
 end
 
+group :development do
```
[the remark this answers is not in the document]
Comment (SolomonHD) [reply Gemfile]: We only need it on boxes we're deploying **from** not **to**.

File: config/role_map.yml [hunk 3]
```diff
@@ -20,4 +20,6 @@ test:
     - leland_himself@example.com
 
 production:
+  archivist:
```
[the remark this answers is not in the document]
Comment (SolomonHD) [reply config/role_map.yml]: Needed for deploying to production or it causes an error.

**Comment by rotated8** [comment]
@SolomonHD I've rebased this and squashed it into one commit. Please take a look and make sure I haven't mangled your work. This can be merged after you leave a review.

**Comment by SolomonHD** [comment]
@rotated8 Everything looks good to me I think we're ready for merging with main branch.

**Pull request merged**